### PR TITLE
add model version

### PR DESCRIPTION
**Pull request by fabianfrz**
requested by @mimugmail  in #393

### CODE REVIEW

File: mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <model>
   <mount>//OPNsense/Rspamd</mount>
   <description>rspamd anti spam filter</description>
+  <version>0.0.0</version>
```
Comment (fichtner) [inline mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: i think this translates to "unversioned"

Comment (fabianfrz) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: 0.0.0 is the default.

Comment (fichtner) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: yes, that's true. 0.0.0 is unversioned, which is the default.

Comment (fabianfrz) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: yes - and no bigger version exists as there is nothing that needs a migration.

Comment (fichtner) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: It's all the same, we don't need this line if we don't use it.

Comment (mimugmail) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: I always thought 1.0.0 is initial stable?

Comment (fabianfrz) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: no, the default is 0.0.0 and any different version will trigger the migration update.

The model version is not visible to the user (only the PLUGIN_VERSION)

Comment (fichtner) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: guys, it's pretty simple: either set this to something not 0.0.0 or remove the change until a migration is needed.

**Comment by fichtner** [comment]
This still needs fixing as reported by @mimugmail 

https://github.com/fabianfrz/plugins/blob/be568fc559118bcba85abbb9ae6970e18df52942/mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/Menu/Menu.xml#L3

File: mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml (hunk 1 above)
Comment (fabianfrz) [reply mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml]: @fichtner than I'll keep it out.

**Comment by fabianfrz** [comment]
@fichtner it's not only rspamd

**Comment by fichtner** [comment]
I have no clue what you are talking about... you have set the name "Rspamd" but the node is called "antispam" so the sort order is wrong...

**Comment by fabianfrz** [comment]
@fichtner #398 - also redis is incorrectly sorted because up/downcase writing of tags also impacts sorting.

**Pull request closed without merging**